src/genetic_demux/freemuxlet/config.vsh.yaml

```diff
@@ -0,0 +1,118 @@
+    - name: "--auxFiles"
+      type: boolean_true
+      description: "Turn on writing auxilary output files"
+    - name: "--verbose"
+      type: integer
+      default: 100
+      description: "Turn on verbose mode with specific verbosity threshold. 0: fully verbose, 100 : no verbose messages."
+    - name: "--doubletPrior"
+      type: double
+      default: 0.5
+      description: "Prior of doublet."
+    - name: "--genoError"
+      type: double
+      default: 0.1
+      description: "Genotype error parameter per cluster."
+    - name: "--bfThres"
+      type: double
+      default: 5.41
+      description: "Bayes Factor Threshold used in the initial clustering."
+    - name: "--fracInitClust"
+      type: double
+      default: 1
+      description: "Fraction of droplets to be clustered in the very first round of initial clustering procedure."
+    - name: "--iterInit"
+      type: integer
+      default: 10
+      description: "Iteration for initial cluster assignment (set to zero to skip the iterations)."
+    - name: "--keepInitMissing"
+      type: boolean_true
+      description: "Keep missing cluster assignment as missing in the initial iteration."
+    - name: "--randomizeSingletScore"
+      type: boolean_true
+      description: "Randomize the singlet scores to test its effect."
+    - name: "--seed"
+      type: integer
+      default: 0
+      description: "Seed for random number (use clocks if not set)."
+    - name: "--capBQ"
+      type: integer
+      default: 20
+      description: "Maximum base quality (higher BQ will be capped)."
+    - name: "--minBQ"
+      type: integer
+      default: 13
+      description: "Minimum base quality to consider (lower BQ will be skipped)."
+    - name: "--groupList"
+      type: string
+      description: "List of tag readgroup/cell barcode to consider in this run. All other barcodes will be ignored. This is useful for parallelized run."
+    - name: "--minTotal"
+      type: integer
+      default: 0
+      description: "Minimum number of total reads for a droplet/cell to be considered."
+    - name: "--minUmi"
+      type: integer
+      default: 0
+      description: "Minimum number of UMIs for a droplet/cell to be considered."
+    - name: "--minSnp"
+      type: integer
+      default: 0
+      description: "Minimum number of SNPs with coverage for a droplet/cell to be considered."
+  - name: "Output"
+    arguments:
+    - name: "--output"
+      alternatives: [ "-o" ]
+      type: file
+      direction: output
+      description: Output directory 
+      example: "freemux/"
+    - name: "--out"
+      type: string
+      description: freemuxlet Output file prefix
+      example: "freemuxlet"
+  resources:
+    - type: bash_script
+      path: script.sh
+    - type: r_script
+      path: summary.R
+  test_resources:
+    - type: bash_script
+      path: test.sh
+    - path: ../../../resources_test/demuxafy_test_data
+platforms:
+  - type: docker
+    image: ubuntu:20.04
```

Comment:
Use an older image?

Reply:
popscle only worked on the older image. There is [a feature request](https://github.com/statgen/popscle/issues/59) already but no update